analytics_data_api/v0/exceptions.py

```diff
@@ -0,0 +1,52 @@
+import abc
+
+
+class BaseError(Exception):
+    """
+    Base error.
+    """
+
+    __metaclass__ = abc.ABCMeta
```

Comment:
I said it poorly, but I think what I meant was that the abc/message/__str__ stuff may be unnecessary because the base Exception class provides that functionality itself. I don't mind leaving this, though.